test/test_helper.bash

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+# Disallow any system-level git-secret pattern configs
+export GIT_CONFIG_NOSYSTEM="true"
```

Comment:
I think `GIT_CONFIG_SYSTEM=""` is the modern (git 2.3.2+) equivalent of this.
This might need `GIT_CONFIG_GLOBAL=""` as well.